CHANGELOG.md

```diff
@@ -5,6 +5,15 @@
 * SourceKitten now requires Xcode 9 and Swift 3.2+ to build.  
   [Norio Nomura](https://github.com/norio-nomura)
 
+* Deprecated `Request.send()`. Please use `Request.failableSend()` instead.  
```

Comment:
I feel like we should instead rename `failableSend()` to just `send()`. Throwing APIs in the Swift stdlib aren't prefixed with `failable`. It made sense to call it that when we had both, but now that we're removing that, I don't think it's appropriate anymore.